core/src/test/scala/cats/derived/show.scala

```diff
@@ -0,0 +1,31 @@
+package cats.derived
+import cats.Show
+import cats.derived.TestDefns.{Foo, Node, Tree, Leaf}
+import MkShow._
+
+class ShowTests extends KittensSuite {
+
+  test("for simple product") {
+    import cats.instances.all._
+    val foo = Foo(42, Option("Hello"))
+    assert(Show[Foo].show(foo) == "Foo(i = 42, b = Some(Hello))")
```

Comment:
Yes thank you @SystemFw, I guess it is not ideal, but it will work for a lot of cases. @milessabin said on gitter, we might be able to provide full derivation by splitting `describe` from `Typeable`. Should I open an issue on this? :)